ui/dashboards/src/components/Panel/PanelActions.tsx

```diff
@@ -50,7 +51,9 @@ export interface PanelActionsProps {
     isPanelViewed?: boolean;
     onViewPanelClick: () => void;
   };
-  queryResults: QueryData[];
+  queryResults: TimeSeriesData | undefined;
+  projectName?: string;
+  panelProps?: Record<string, unknown>;
```

Comment:
We can use the `PanelContentProps` interface here. this contains already the pluginKind and the data so the plugin can use it to build actions.